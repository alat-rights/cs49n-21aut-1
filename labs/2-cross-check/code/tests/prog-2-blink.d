prog-2-blink.o: prog-2-blink.c ../rpi.h
