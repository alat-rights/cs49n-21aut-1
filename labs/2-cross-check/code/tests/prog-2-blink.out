TRACE: calling pi code
TRACE: GET32(0x20200008) = 0x66334873
TRACE: PUT32(0x20200008) = 0x66334871
TRACE: GET32(0x20200008) = 0x66334871
TRACE: PUT32(0x20200008) = 0x66334849
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: PUT32(0x20200028) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x20200028) = 0x100000
TRACE: PUT32(0x2020001c) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: PUT32(0x20200028) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x20200028) = 0x100000
TRACE: PUT32(0x2020001c) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: PUT32(0x20200028) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x20200028) = 0x100000
TRACE: PUT32(0x2020001c) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: PUT32(0x20200028) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x20200028) = 0x100000
TRACE: PUT32(0x2020001c) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: PUT32(0x20200028) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x20200028) = 0x100000
TRACE: PUT32(0x2020001c) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: PUT32(0x20200028) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x20200028) = 0x100000
TRACE: PUT32(0x2020001c) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: PUT32(0x20200028) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x20200028) = 0x100000
TRACE: PUT32(0x2020001c) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: PUT32(0x20200028) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x20200028) = 0x100000
TRACE: PUT32(0x2020001c) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: PUT32(0x20200028) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x20200028) = 0x100000
TRACE: PUT32(0x2020001c) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: PUT32(0x20200028) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: PUT32(0x20200028) = 0x100000
TRACE: PUT32(0x2020001c) = 0x200000
TRACE: delaying 3000000 cycles
TRACE: pi exited cleanly: 7 calls to random
